datasketches-memory-java8-tests/src/test/java/org/apache/datasketches/memory/test/Buffer2Test.java

```diff
@@ -406,6 +408,14 @@ public void testIllegalFill() {
     wbuf.fill((byte)0);
   }
 
+  @Test
+  public void checkWritableWrap() {
+    ByteBuffer bb = ByteBuffer.allocate(16);
+    WritableBuffer buf = WritableBuffer.writableWrap(bb, ByteOrder.nativeOrder(), null);
+    buf = WritableBuffer.writableWrap(bb, ByteOrder.nativeOrder(), new DefaultMemoryRequestServer());
+    assertNotNull(buf);
```

Comment:
Is it necessary to check that. buf is not null after line 414 as well?

Reply:
Yes, I should have checked twice, but it is very unlikely that one would be null and the other not.